agent/replacecall/src/main/java/randoop/instrument/ReplaceCallAgent.java

```diff
@@ -0,0 +1,207 @@
+package randoop.instrument;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.lang.instrument.Instrumentation;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import plume.EntryReader;
+import plume.Option;
+import plume.Options;
+import randoop.MethodReplacements;
+
+/**
+ * The replacecall javaagent applies the {@link CallReplacementTransformer} to replace calls in
+ * loaded classes by calls to alternate implementations.
+ *
+ * <p>The transformer applies method call replacements as specified in either the default or a
+ * user-provided replacement file. (See the <a
+ * href="https://randoop.github.io/randoop/manual/index.html#replacecall">replacecall user
+ * documentation</a> the file format.) Default replacements are given in an internal resource file
+ * {@code "default-replacements.txt"}. User replacements are then loaded using the {@code
+ * --map-calls} command-line argument. A user replacement may override a default replacement.
+ *
+ * <p>The classes of packages listed in the resource file {@code "default-load-exclusions.txt"} are
+ * excluded from transformation.
+ */
+public class ReplaceCallAgent {
+
+  /** Run the replacecall agent in debug mode. */
+  @SuppressWarnings("WeakerAccess")
+  @Option("print debug information")
+  public static boolean debug = false;
+
+  @SuppressWarnings("WeakerAccess")
+  @Option("directory name where debug logs are written")
```

Comment:
Add Javadoc.  For example, it seems that null means the current directory.  (I would have guessed that null means no debug logging.)

Also add Javadoc for debugPath, and explain its relationship to this variable.  Why not have just one variable, a Path?  (If it is because `@Option` does not handle Path, that is easily remedied.)